src/bindings/js/node/include/node_wrap.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <napi.h>
+#include <openvino/openvino.hpp>
+
+class NodeWrap: public Napi::ObjectWrap<NodeWrap>{
+    public:
+        /**
+        * @brief Constructs NodeWrap from the Napi::CallbackInfo.
+        * @param info contains passed arguments. Can be empty.
+        */
+        static Napi::Object Init(Napi::Env env, Napi::Object exports);
+        NodeWrap(const Napi::CallbackInfo& info);
+        
+    
+    private:
+        static Napi::FunctionReference constructor;
+        ov::Node* node;
```

Comment:
Since the ov::node is managed by the OpenVINO library, there is no need for us to handle its memory. However, if we wish to manage the object's lifetime then it would be store directly (without pointer).